### PR TITLE
Add distributed copy command.

### PR DESCRIPTION
**Pull request by cheng-chang**
`alluxio fs distributedCp src dst` will instruct job workers to copy files under src to dst in parallel, the parallel copy happens at file level, not block level.

### CODE REVIEW

File: core/base/src/main/java/alluxio/exception/ExceptionMessage.java [hunk 1]
```diff
@@ -13,9 +13,8 @@
 
 import com.google.common.base.Preconditions;
 
-import java.text.MessageFormat;
```
Comment (ZacBlanco) [inline core/base/src/main/java/alluxio/exception/ExceptionMessage.java]: Could we put this import back to where it was?

File: shell/src/main/java/alluxio/cli/fs/command/DistributedCpCommand.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+import alluxio.client.file.FileSystemContext;
+import alluxio.client.job.JobGrpcClientUtils;
+import alluxio.exception.AlluxioException;
+import alluxio.exception.status.InvalidArgumentException;
+import alluxio.job.migrate.MigrateConfig;
+
+import org.apache.commons.cli.CommandLine;
+
+import javax.annotation.concurrent.ThreadSafe;
+import java.io.IOException;
```
Comment (ZacBlanco) [inline shell/src/main/java/alluxio/cli/fs/command/DistributedCpCommand.java]: This should be in its own import group above javax

Comment (cheng-chang) [reply shell/src/main/java/alluxio/cli/fs/command/DistributedCpCommand.java]: done

**Comment by cheng-chang** [comment]
@ZacBlanco PTAL

**Comment by cheng-chang** [comment]
jenkins, test this please

**Pull request merged**